src/api/app/controllers/webui/users/notifications_controller.rb

```diff
@@ -89,11 +89,6 @@ def set_counted_notifications
     @counted_notifications['appealed_decisions'] = @notifications.for_appealed_decisions.count
   end
 
-  def update_counted_notifications
```

Comment:
Please do not drop it, see https://github.com/openSUSE/open-build-service/pull/16273

Reply:
Right. I removed the commit which dropped it.